toolkit/tools/imagecustomizer/docs/clone-rpm-repo.md

```diff
@@ -0,0 +1,177 @@
+# Cloning an RPM repo
```

Comment:
Do we need a link from the landing page?

Reply:
Added a link from `--rpm-source` in `cli.md`.